ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,199 @@
+
+Actions are the individual activities being executed in the workflow and are represented by a rectangle. Actions serve as building blocks, which are chained together through places, to configure a workflow.
+
+All actions contain a certain, predefined number of input and output ports, at least 1 of each, which can be mandatory or optional. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports). For an action to be executable, all **connected** inputs and outputs, mandatory or optional, must be fed with data.
+
+The actions available to create workflows depend on the modules installed in the system. Every API end point of the modules automatically becomes available as an action and can be used in any workflow. However, you can also build your own actions independent from the API.
+
+[comment]: <> (Ich finde die Benennung API endpoint etwas verwirrend hier, denn wir haben soeben über start/end point als start/end place gesprochen. Alternative Bennenung?)
+
+It must be differentiated between a workflow action and a process action. A workflow action acts as a building block to create a workflow to model a company process, whereas a process action is the actual execution of the configured workflow action. As such, a process action can be successful or erroneous depending on its execution. If an error occurs, a process action can be retried if the underlying cause has been fixed, which results in a new process action. If multiple tokens move through the workflow, every token is processed in its own process action. As a result, any workflow action can be associated with multiple process actions, depending of the number of tokens running through it.
+
+[commment]: <> (Evtl. kürzer/weniger Details?)
+
+When adding a new action, all compatible input and output ports are included. Mandatory input ports are already displayed and linked to a place, whereas optional ones are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button.
+
+
+When clicking an action, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+
+The action settings side bar is located to the right of the workspace and allows to define the action settings. The following fields are displayed:
+
+  - (Action label and name)  
+  The action label is an action identification tag and can be defined by user. The action name is a system wide predefined denomination of the activity to be performed and is read-only.
+
+  - *Key*  
+   Action key. This field is read-only. The key is required for API access and must be system wide unique.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
```

Comment:
```suggestion
  Action label defined by the user. It can be used to specify or simplify the action name. When editing the action label, it is changed both, at the top of the settings side bar and in the action rectangle in the diagram.
```